radixdlt-core/radixdlt/src/main/java/com/radixdlt/application/NodeApplication.java

```diff
@@ -109,8 +109,9 @@ private void processRequest(NodeApplicationRequest request) {
 			var txBuilder = radixEngine.construct(request.getRequest().feePayer(REAddr.ofPubKeyAccount(self)));
 			var txn = txBuilder.signAndBuild(hashSigner::sign);
 			var mempoolAdd = request.completableFuture()
-				.map(f -> MempoolAdd.create(txn, f))
+				.map(future -> MempoolAdd.create(txn, future))
```

Comment:
Would prefer to not have arbitrary decoration changes in non-relevant classes. But fine for now.